### PR TITLE
bump the tunnel images.

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -96,7 +96,9 @@
       "versions": [
         "v1.9.2-v3.0.18",
```
Comment (adilliadil) [inline vhdbuilder/packer/components.json:97]: Do we need the previous versions?

Comment (andyliuliming) [reply vhdbuilder/packer/components.json]: deleted now.

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -108,9 +109,8 @@
     {
       "downloadURL": "mcr.microsoft.com/aks/hcp/tunnel-openvpn:*",
       "versions": [
-        "1.0.8",
-        "1.0.10",
-        "1.0.11"
+        "1.0.11",
+        "master.0326.5"
```
Comment (xuto2) [inline vhdbuilder/packer/components.json:113]: what is this?

**Review by xuto2 — APPROVED** [review]
I have no concern as long as you verify the removed images are no longer used by toggle and the new format is the permanent change. 

Feel free to merge by yourselves

**Review by adilliadil — APPROVED** [review]

**Pull request merged**